### PR TITLE
README.md: more accurate methodWhitelist example

### PR DESCRIPTION
**Pull request by pepakriz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ In some cases, you may want to ignore exception-related errors as per class basi
 parameters:
 	exceptionRules:
 		methodWhitelist:
-			PHPUnit\Framework\TestCase: '#^(setup|setupbeforeclass|teardown|teardownafterclass|test.+)$#i'
+			PHPUnit\Framework\TestCase: '#^(setup|setupbeforeclass|teardown|teardownafterclass|test.*)$#i'
```
Comment (marcospassos) [inline README.md]: should the method `public function test()` be considered a test?

Comment (pepakriz) [reply README.md]: Yes, I've tested it

Comment (marcospassos) [reply README.md]: I'm not sure about it, but if it does, the `.*` is not needed as we're already matching methods starting with at least one of these prefixes.

Comment (marcospassos) [reply README.md]: (the `$` should also be left out)

Comment (marcospassos) [reply README.md]: ok, so the more correct would be `#^(test|setup|setupbeforeclass|teardown|teardownafterclass)#i`

Comment (marcospassos) [reply README.md]: #^test|(setup|setupbeforeclass|teardown|teardownafterclass)$#i

Comment (marcospassos) [reply README.md]: The more correct would be `#^test|^(setup|setupbeforeclass|teardown|teardownafterclass)$#i`

Comment (marcospassos) [reply README.md]: @pepakriz how could we provide a preset configuration so that dev doesn't need to repeat it on every project?

Comment (pepakriz) [reply README.md]: Your example is more optimized but less readable for me. I don't care.

Comment (pepakriz) [reply README.md]: I'm not a fan of universal presets. Every project is unique and needs different setup and rules. But I'm open to suggestions

Comment (marcospassos) [reply README.md]: the performance penalty over the `.*` cannot be ignored. How about `#^(test|(setup|setupbeforeclass|teardown|teardownafterclass)$)#i`?

Comment (pepakriz) [reply README.md]: Much better, I'll change it

**Review by marcospassos — APPROVED** [review]

**Pull request merged**